src/index.html

```diff
@@ -1,30 +1,47 @@
 <!DOCTYPE html>
-<html>
+
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
   <head>
-    <title>Fit Lit</title>
-    <meta charset="UTF-8">
+    <meta charset="utf-8">
+    <title>FITLIT</title>
     <link rel="stylesheet" href="styles.css">
     <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
-    <title>Activity Tracker</title>
   </head>
+  <html>
   <body>
-    <header>
-      <h1>Activity Tracker</h1>
+    <nav>
       <h3>Welcome <span class="user-first-name-js"></span>!</h3>
-    </header>
-    <main>
-      <section class="user-card user-card-js">
-        <p>Name: <span class="user-full-name-js"></span></p>
-        <p>ID: <span class="user-id-js"></span></p>
-        <p>Address: <span class="user-address-js"></span></p>
-        <p>email: <span class="user-email-js"></span></p>
-        <p>Stride Length: <span class="user-stride-length-js"></span></p>
-        <p>Daily Step Goal: <span class="user-daily-step-goal-js"></span></p>
-        <p>Other People's Average Step Goal: <span class="step-goal-comparison-js"></span></p>
-        <p>Friends: <span class="user-friends-js"></span></p>
+    </nav>
```

Comment:
These are the fancy widget tiles that make our DOM look a little bit better than the previous iteration.  

Each tile now has its own data that is being populated by using innerHTML in our scripts.js file.